compiler/src/pkg/manifest.rs

```diff
@@ -1,12 +1,50 @@
+use crate::config::Config;
 use crate::pkg::version::Version;
 use blake3;
 use std::fmt;
-use std::fs::{File, OpenOptions};
+use std::fs::{read_dir, File, OpenOptions};
 use std::io::{BufRead, BufReader, Read, Write};
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
 pub const MANIFEST_FILE: &str = "inko.pkg";
 
+pub fn all_manifests(
+    config: &Config,
+) -> Result<Vec<(Manifest, PathBuf)>, String> {
+    let mut manifests = Vec::new();
+    let mut paths = Vec::new();
+    let dependencies = &config.dependencies;
+
+    if let Ok(dependencies) = read_dir(dependencies) {
+        for dependency in dependencies.filter_map(Result::ok) {
```

Comment:
Does this mean that if a directory can't be read, it will be ignored? If so, I think it's better to be explicit/careful and produce an error if the directory can't be read.

Reply:
Indeed. Updated to propagate errors whatever the `ReadDir` iterator returns.